test/test_torch.py

```diff
@@ -5994,9 +5994,9 @@ def test_grad_scale_will_not_overflow(self, device):
         assert scaler._scale != float("inf") and scaler._scale != float("nan")
 
     @onlyNativeDeviceTypes
-    @skipIfTorchDynamo("https://github.com/pytorch/pytorch/issues/123238")
     @dtypes(*floating_types_and(torch.bfloat16, torch.half))
     @precisionOverride({torch.half : 1e-3, torch.bfloat16 : 1e-3})
+    @skipIfTorchDynamo("https://github.com/pytorch/pytorch/issues/123238")
```

Comment:
Move this to be in common_optimizers.py (we have all our dynamo skips consolidated there). Same with the precision override.

Reply:
I totally removed `test_fused_adam_and_adamw` and use `test_fused_matches_forloop` to test them. I followed your suggestion with `TensorTracker` with it and so modified `_test_derived_optimizers`